File: Components/WinForms/DevExpress.VideoRent.Win/CS/DevExpress.VideoRent.Win/ModulesReports/TheMostProfitableGenresList.cs
using DevExpress.VideoRent.Reports;
namespace DevExpress.VideoRent.Win.ModulesReports {
    public partial class TheMostProfitableGenresList : ReportModuleBase {
        public TheMostProfitableGenresList() {
            InitializeComponent();
        }

        protected override ReportBase CreateReport() {
            return new TheMostProfitableGenresListReport();
        }
    }
}

File: Components/WinForms/DevExpress.VideoRent.Win/VB/DevExpress.VideoRent.Win/ModulesReports/TheMostProfitableGenresList.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.VideoRent.Reports
Namespace DevExpress.VideoRent.Win.ModulesReports
	Partial Public Class TheMostProfitableGenresList
		Inherits ReportModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub

		Protected Overrides Function CreateReport() As ReportBase
			Return New TheMostProfitableGenresListReport()
		End Function
	End Class
End Namespace
